react_django/react-src/src/components/student/hpProfileView/hpProfileView.css

```diff
@@ -0,0 +1,214 @@
+/* STUDENT SETTING CSS */
```

Comment:
Sorry I'm just now getting this, it totally slipped through the net. This just looks like a mis-type to me (should be Hiring Partner CSS, right?), and I want style purposes clearly documented before putting them on master (so any dev can use them).

Reply:
You're right. Editing in another repo and missed some description differences. All changed and ready for review.